x-pack/plugins/cases/server/client/client.ts

```diff
@@ -36,6 +41,13 @@ export class CasesClient {
     return this._userActions;
   }
 
+  public get subCases() {
+    if (!ENABLE_CASE_CONNECTOR) {
+      throw Boom.badRequest('The case connector feature is disabled');
```

Comment:
I think we shouldn't throw a boom error but a `CasesError`. It is up to the consumer of our client to decide what to do with an error. We shouldn't be opinionated about it as the cases client can be used outside a route.

Reply:
Good point, I'll switch it to an `Error` 👍